docs/quickstarts/aws.md

```diff
@@ -0,0 +1,207 @@
+# Lokomotive AWS installation guide
+
+## Contents
+
+* [Introduction](#introduction)
+* [Requirements](#requirements)
+* [Step 1: Install Lokoctl](#step-1-install-lokoctl)
+* [Step 2: Set up a working directory](#step-2-set-up-a-working-directory)
+* [Step 3: Set up credentials from environment variables](#step-3-set-up-credentials-from-environment-variables)
+* [Step 4: Define cluster configuration](#step-4-define-cluster-configuration)
+* [Step 5: Create Lokomotive cluster](#step-5-create-lokomotive-cluster)
+* [Verification](#verification)
+* [Cleanup](#cleanup)
+* [Troubleshooting](#troubleshooting)
+* [Conclusion](#conclusion)
+* [Next steps](#next-steps)
+
+## Introduction
+
+This quickstart guide walks through the steps needed to create a Lokomotive cluster on AWS with
+Flatcar Container Linux using Route53 as the DNS provider.
+
+By the end of this guide, you'll have a production-ready Kubernetes cluster running on AWS.
+
+## Requirements
+
+* Basic understanding of Kubernetes concepts.
+* AWS account and IAM credentials.
+* AWS Route53 DNS Zone (registered Domain Name or delegated subdomain).
+* Terraform v0.12.x and [terraform-provider-ct](https://github.com/poseidon/terraform-provider-ct)
+  installed locally
+* An SSH key pair for management access.
+* `kubectl` installed locally to access the Kubernetes cluster.
+
+## Steps
+
+### Step 1: Install lokoctl
+
+Lokoctl is a command-line interface for Lokomotive.
+
+To install `lokoctl`, follow the instructions in the [lokoctl installation](../installer/lokoctl.md)
+guide.
+
+### Step 2: Set up a working directory
+
+
+It's better to start fresh in a new working directory, as the state of the cluster is stored in this
+directory.
+
+This also makes the cleanup task easier.
+
+
+```console
+mkdir -p lokomotive-infra/myawscluster
+cd lokomotive-infra/myawscluster
+```
+
+### Step 3: Set up credentials from environment variables
+
+The AWS credentials file can be found at `~/.aws/credentials` if you have set up and configured AWS
+CLI before.  If you want to use that account, you don't need to specify any credentials for lokoctl.
+
+You can also take any other credentials mechanism used by the AWS CLI but environment variables may
+be the safest option. Either prepend them when starting lokoctl or export each of them once in the
+current terminal session:
+
+```console
+AWS_ACCESS_KEY_ID=EXAMPLEID AWS_SECRET_ACCESS_KEY=EXAMPLEKEY lokoctl ...
+```
+or
+
+```console
+export AWS_ACCESS_KEY_ID=AKIAIOSFODNN7EXAMPLE
+export AWS_SECRET_ACCESS_KEY=wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
+export AWS_DEFAULT_REGION=us-east-1
+```
+
+### Step 4: Define cluster configuration
+
+To create a Lokomotive cluster, we need to define a configuration.
+
+A [production-ready configuration](../../examples/aws-production) is already provided for ease of
+use.  Copy the example configuration to the working directory and modify accordingly.
```

Comment:
```suggestion
use. Copy the example configuration to the working directory and modify accordingly.
```

Reply:
Also, I wouldn't suggest, that the user should modify this configuration file ("modify accordingly"). I would say something like:
> If you would like to modify default configuration, see components and platform documentation documents.

The intention is, that user should be able to use this configuration file by just dropping in correct `lokocfg.vars` file.

Reply:
This being a production cluster configuration example , the user is bound to modify it according to their.

Might as well say it and then point them to different documentation for their reference.